controller/space.go

```diff
@@ -312,7 +312,8 @@ func ConvertSpace(request *goa.RequestData, p *space.Space, additional ...SpaceC
 	relatedIterationList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/iterations", p.ID.String()))
 	relatedAreaList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/areas", p.ID.String()))
 	relatedCodebasesList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/codebases", p.ID.String()))
-
+	relatedWorkItemList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/workitems", p.ID.String()))
```

Comment:
I don't like that we have these hardcoded routes in many places, I'll work on a improvement.